WPMedia/fusion-news-theme-blocks
# THEMES-761: Identity Block | Social Sign On

## Description

This PR adds in the Social Sign On block, as a part of the Identity blocks.

## Jira Ticket

- [THEMES-761](https://arcpublishing.atlassian.net/browse/THEMES-761)

## Acceptance Criteria

The Identity Social Sign On block is converted to Themes V2

## Test Steps

1. Checkout this branch `git checkout THEMES-761-social-sign-on-no-package`
2. Checkout the feature pack branch `git checkout THEMES-761`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
4. Create or edit a page and add the "Identity Log In - Arc Block", "Identity Login Links - Arc Block", and "Identity Social Sign On - Arc Block".
5. The output should match the mock ups with one exception.
    * The styling and widths of the Google/Facebook buttons are controlled by their respective libraries, so the background color, text, and icon placement we have little to no control over. I've chosen the options that get as close to the designs as possible.

## Dependencies or Side Effects

- Feature pack pr for local development: https://github.com/WPMedia/arc-themes-feature-pack/pull/467

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-761]: https://arcpublishing.atlassian.net/browse/THEMES-761?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
